test/setup-deps/default.nix

```diff
@@ -15,10 +15,10 @@ let
   };
 
   packages = project.hsPkgs;
-in recurseIntoAttrs (if stdenv.hostPlatform.isWindows
+in recurseIntoAttrs (if stdenv.buildPlatform != stdenv.hostPlatform
```

Comment:
Conceptually we should be able to execute for `musl`. And for others if we have `test-wrappers` available.